go.mod

```diff
@@ -2,8 +2,9 @@ module github.com/terraform-providers/terraform-provider-azurerm
 
 require (
 	github.com/Azure/azure-sdk-for-go v46.4.0+incompatible
-	github.com/Azure/go-autorest/autorest v0.11.10
+	github.com/Azure/go-autorest/autorest v0.11.3
 	github.com/Azure/go-autorest/autorest/date v0.3.0
+	github.com/Azure/go-autorest/autorest/validation v0.3.0
```

Comment:
Can we rebase this branch/revert these and re-run `go mod vendor && go mod tidy` - which should re-upgrade these? This is necessary since it fixes a potential security vulnerability in the Azure SDK/go-autorest